vendor/github.com/golang/glog/glog.go

```diff
@@ -676,7 +676,10 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		}
 	}
 	data := buf.Bytes()
-	if l.toStderr {
```

Comment:
Let's drop the changes to vendor/github.com/golang/glog/* the change below causes problems